tlslite/utils/python_aesgcm.py

```diff
@@ -4,7 +4,8 @@
 """Pure-Python AES-GCM implementation."""
 
 from .aesgcm import AESGCM
-from .rijndael import rijndael
+from .rijndael import Rijndael
+
```

Comment:
Why the extra line added here?

Reply:
pep-8; top level objects should be separated by two empty lines

Reply:
Ah, ok, then in other cases (other source files) there is just one empty line after top level object, I believe.

Reply:
yes, there is a lot of stylistic errors, I'm trying to fix them when I work on files, but I have other priorities

this PR is only a side effect of looking for the issue fixed in #297 (I originally thought that the issue was in AES code), otherwise I wouldn't have worked on it at all

Reply:
OK, sure